x-pack/plugins/discover_log_explorer/common/datasets/models/dataset.ts

```diff
@@ -26,16 +33,37 @@ export class Dataset {
     };
   }
 
+  getFullTitle(): string {
+    return this.parentIntegration?.name
+      ? `[${this.parentIntegration.name}] ${this.title}`
+      : this.title;
+  }
+
   toDataviewSpec(): DataViewSpec {
     // Invert the property because the API returns the index pattern as `name` and a readable name as `title`
     return {
       id: this.id,
-      name: this.title,
-      title: this.name,
+      name: this.getFullTitle(),
+      title: this.name as string,
     };
   }
 
-  public static create(dataset: DatasetType, parentIntegration?: IntegrationType) {
+  toPlain() {
+    return {
+      name: this.name,
+      title: this.title,
+    };
+  }
+
+  public static create(dataset: DatasetDeps, parentIntegration?: IntegrationBase) {
     return new Dataset(dataset, parentIntegration);
   }
+
+  public static createAllLogsDataset() {
+    return new Dataset({
+      name: 'logs-*-*' as IndexPattern,
+      title: 'All log datasets',
```

Comment:
**question:** shall we translate this title or can it be a static value as the other dataset names?